Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -3174,6 +3174,12 @@ public IWebElement LocateElement(Act act, bool AlwaysReturn = false, string Vali
                             SwitchFrame(currentPOMElementInfo);
                         }
                         elem = LocateElementByLocators(currentPOMElementInfo.Locators);
+
+                        if (pomExcutionUtil.AutoUpdateCurrentPOM() != null)
```

Comment:
if element is not null then no need to update the pom. So we can add check elem== null then only go inside